src/ui/components/Button/Button.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { theme } from '../../../shared/theme';
+import ButtonUrl from '../../../public/img/silverButton.png';
+
+const BUTTON_HEIGHT_SIZE = '8rem';
+const BUTTON_WIDTH_SIZE = '21.25rem';
+const BUTTON_FONT_SIZE = '22px';
+
+/**
+ * @param {{ children: React.ReactNode, type: 'normal' | 'selected', onSelect: () => void }} props
+ */
+const Button = ({ children, type, onSelect }) => {
+  const onClick = React.useCallback(() => {
+    if (type === 'normal') {
+      onSelect();
+    }
+  }, [type, onSelect]);
```

Comment:
Tu trzeba raczej zmienić trochę logikę. Coś takiego było u mnie, bo `LevelButton` to w sumie taki odpowiednik `<input type="radio">` albo `<select>`, ale ten tu to odpowiednik zwykłego `<button>` więc myślę, że zwykły `onClick` wystarczy. Tak samo warianty bym raczej nazwał w stylu `primary`/`secondary` albo `main`/`normal` bo bardziej temu odpowiadają niż temu czy przycisk jest "wybrany".